port/unix/omrsysinfo.c

```diff
@@ -4557,6 +4575,15 @@ omrsysinfo_cgroup_subsystem_iterator_next(struct OMRPortLibrary *portLibrary, st
 	return rc;	
```

Comment:
`*printUnits = FALSE;` just above this - can this just ensure that  `metricElement->units` is null?  And then `printUnits` can be removed?